### DISCUSSION TITLE
Singularity/Docker OpenMS image, with 3rd party softwares already built ?

### DISCUSSION THREAD

**Discussion by polklin** [OpenMS/OpenMS#6455] [Q&A] [answered]
Hello !

I am pretty new new to OpenMS and started to play with it thanks to a singularity image (openms:2.7.0--h15c4245_1)  from https://depot.galaxyproject.org/singularity/.

I was wondering if 3rd party programs (such as Comet and MSGF+ search engines) are installed in this image and if yes where ? 
If not, do you know if there are existing docker/singularity images where OpenMS is installed, with search engines and Percolator already built ?

Thank you very much for your help !
Best,
Paul

**Comment by jpfeuffer** [comment, marked as answer]
Hi Paul,

have you checked openms-thirdparty? It may not always be up-to-date but we try to list all third-party versions there that exist on bioconda and are compatible with the openms version at the time of the release.

Best
Julianus

**Reply by polklin** [reply]
Indeed that is very cool !
Thank you very much for your quick reply :)
